packages/nextra/src/loader.ts

```diff
@@ -208,39 +231,55 @@ export default MDXContent`
       // Remove the only `index` route
       .replace(/^index$/, '')
 
+  const stringifiedPageOpts = JSON.stringify(pageOpts)
+  const pageOptsChecksum = hashFnv32a(stringifiedPageOpts)
```

Comment:
To avoid calling function on production
```suggestion
  const pageOptsChecksum = process.env.NODE_ENV !== 'production' && hashFnv32a(stringifiedPageOpts)
```